sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -95,13 +97,39 @@ namespace Azure { namespace Core { namespace Http {
     std::unique_ptr<RawResponse> Send(Context const& ctx, Request& req);
   };
 
+  /**
+   * @brief The options for the #TransportPolicy.
+   *
+   */
+  struct TransportPolicyOptions
```

Comment:
Do we usually put 'Policy' in the struct name here or should this be TransportOptions.